### ISSUE TITLE
LAN devices

### ISSUE THREAD

**Issue by unpack8866** [PaulAnnekov/home-assistant-padavan-tracker#7]
Can increase the detection of LAN devices?

**Comment by PaulAnnekov** [comment]
I don't think it will be more reliable, than [nmap tracker](https://www.home-assistant.io/components/nmap_tracker/). But if anybody think it will, he can make a contribution and I will merge it.

**Comment by unpack8866** [comment]
Trying to use the namp tracker, my device status is constantly changing online and offline, so I hope you can increase the detection of LAN devices. Because the other AP access point devices in the LAN will be displayed as LAN devices in padavan, thank you!

**Comment by PaulAnnekov** [comment]
I will be glad to see a PR. I don't have a time right now to implement this.
